### PR TITLE
[lxd] Add logs from the snap

### PR DESCRIPTION
**Pull request by arif-ali**
---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [x] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [x] Is the subject and message clear and concise?
- [x] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [x] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [ ] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?

### CODE REVIEW

**Review by pmoravec — APPROVED** [review]
ACK to the change itself, please follow DCO and have in the commit message "Closes: #3234" or similar.

File: sos/report/plugins/lxd.py [hunk 1]
```diff
@@ -44,6 +44,17 @@ def setup(self):
         snap_list = self.exec_cmd('snap list lxd')
         if snap_list["status"] == 0:
             self.add_cmd_output("lxd.buginfo", pred=lxd_pred)
+
+            if not self.get_option("all_logs"):
+                self.add_copy_spec([
+                    '/var/snap/lxd/common/lxd/logs/*.log',
+                    '/var/snap/lxd/common/lxd/logs/*/*.log',
+                    '/var/snap/lxd/common/lxd/logs/*/*.conf',
```
Comment (pmoravec) [inline sos/report/plugins/lxd.py:52]: Hm... when re-reading the particular paths to collect: are not config files important enough to be collected every time? They are often older than logfiles so when logs are over size limit, sos report would collect newest logs but not configs (when --all-lags is *not* used).

Does not it deserve to mention configs as an extra `copy_path` in the `else` branch? (or what do config files do in a `log` directory..? :-o)

Comment (arif-ali) [reply sos/report/plugins/lxd.py:52]: The .conf files would be collected with the below line in `else` as seen in https://paste.ubuntu.com/p/n3PYq46XBS/

happy to separate it out, if that is preferred

Comment (pmoravec) [reply sos/report/plugins/lxd.py:52]: If you know the whole `/var/snap/lxd/common/lxd/logs/**` in `else` branch will always be below default size limit, no need to separate. If you are unsure and need to collect configs while optionally truncate older logs, let separate the configs.

Comment (arif-ali) [reply sos/report/plugins/lxd.py:52]: I think I'll separate it, I think that makes more sense

**Review by pmoravec — APPROVED** [review]

**Review by TurboTurtle — APPROVED** [review]

**Pull request merged**